src/routes/GameScreen.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View } from 'react-primitives';
+import { ActivityIndicator } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { game, move } from '../services/GameService';
+
+export default class GameScreen extends Component {
+    static navigationOptions = ({ navigation }) => ({
+        title: `Game #${navigation.state.params.game.id}`,
+    });
+
+    static propTypes = {
+        navigation: PropTypes.object.isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isLoading: true,
+            id: -1,
+            token: '',
+            currentGrid: null,
+            turn: -1,
+            isWinner: null,
+        };
+
+        this.requestGame = this.requestGame.bind(this);
+        this.requestMove = this.requestMove.bind(this);
+    }
+
+    requestGame() {
```

Comment:
Also instead of using `then`, you could make the `requestGame` function `async` and use `await`.

```js
requestGame = async () => {
    const res = await game()(id, token);

    this.setState({
        isLoading: false,
        id,
        token,
        currentGrid: res.currentPlayer.currentGrid,
        turn: res.currentPlayer.turn,
        isWinner: res.winner !== null,
    });
}
```